frontend/src/components/AccountReceiveTableWithdrawConfirmation.vue

```diff
@@ -164,7 +164,7 @@ export default defineComponent({
     if (isNativeToken) {
       formattedAmount = round(formatUnits(amount, decimals), numDecimals); // amount being withdrawn, rounded
     } else {
-      formattedAmount = roundTokenAmount(amount, props.activeFee.token);
+      formattedAmount = humanizeTokenAmount(amount, props.activeFee.token);
     }
```

Comment:
Why do we still need the `if` conditional for the native token everywhere, instead of simplifying it all to `formattedAmount = humanizeTokenAmount(amount, props.activeFee.token);`? There's a few parts that do 

```javascript
if (isNativeToken) {
  formattedAmount = round(formatUnits(...))
} else {
  formattedAmount = humanizeTokenAmount(...)
}
```

which is what I'm referring to

Reply:
Great point, I think you're right that this can be simplified. Nice